Jenkinsfile

```diff
@@ -10,9 +10,22 @@ pipeline {
 	}
 	tools {
 		maven 'apache-maven-latest'
-		jdk 'openjdk-jdk21-latest'
+		jdk 'openjdk-jdk22-latest'
 	}
 	stages {
+		stage('javac specific tests') {
+			steps {
+				sh """#!/bin/bash -x
+					mvn install -DskipTests
+					mvn verify --batch-mode -f org.eclipse.jdt.core.tests.javac --fail-at-end -Ptest-on-javase-22 -Pbree-libs -Papi-check -Djava.io.tmpdir=$WORKSPACE/tmp -Dproject.build.sourceEncoding=UTF-8 -Dcbi-ecj-version=99.99
+"""
+			}
+			post {
+				always {
+					junit 'org.eclipse.jdt.core.tests.java/target/surefire-reports/*.xml'
+				}
+			}
+		}
```

Comment:
You'll most likely need to run it as a 2nd stage, after the existing one.
The `mvn install` can be skipped as it would already be performed by the previous stage.
And I also wonder whether the system properties to enable DOM-first & Javac should be repeated here.

Reply:
I think I will need to reinstall if I want all the bundles installed. I'm pretty sure I need them all installed, since none of them will be built in the reactor, because I'm running only the pom for the tests bundle, so it will need to pull the artifacts from the local repo instead (if I understand the Maven concepts correctly). I've moved the step afterwards.